README.md

```diff
@@ -1034,6 +1034,16 @@ find . -type f -empty -delete
 find . -type f | wc -l
 ```
 
+##### List all files with name [FILE] under /
```

Comment:
searching through on entire system can be very resource-intensive and time-consuming, and should be done with caution. I tend not include this example here, as previous examples show how to search file with name.